snuba/settings/__init__.py

```diff
@@ -22,6 +22,7 @@
 
 DEFAULT_DATASET_NAME = "events"
 DISABLED_DATASETS: Set[str] = set()
+SKIP_HEALTH_CHECK_DATASETS: Set[str] = set()
```

Comment:
Following up on this thought, I imagine that a vendor-ed/third-party managed ClickHouse cluster would have different expectations for what kind of metrics we emit to Datadog, how much we support diagnostic logging, etc.

If the profiling team is doing their own cluster management, we probably shouldn't be subject to their alerting.

Reply:
I don't think a vendor-ed system is what we are building. They are doing it for now, but then their cluster will become part of the production infrastructure like others. This will be used for the experimental phase till we do not have a proper separation of the snuba environment in production vs experimental.